src/Security/Passwords.php

```diff
@@ -46,9 +45,7 @@ public static function hash($password, array $options = NULL)
 	 */
 	public static function verify($password, $hash)
 	{
-		return preg_match('#^\$2y\$(?P<cost>\d\d)\$(?P<salt>.{22})#', $hash, $m)
-			&& $m['cost'] >= 4 && $m['cost'] <= 31
-			&& crypt($password, $hash) === $hash;
+		return preg_match('#^\$2y\$\d\d\$.{22}#', $hash) && password_verify($password, $hash);
```

Comment:
@dg Maybe remove the `preg_match` entirely? It breaks currently one test but for practical purposes it does not really matter.